Use the date from debian/changelog instead of the current date in generated
documentation. This allows support for reproducible builds.
Index: iputils/doc/Makefile
===================================================================
--- iputils.orig/doc/Makefile
+++ iputils/doc/Makefile
@@ -1,5 +1,6 @@
 SHELL:=/bin/bash #required for pushd and popd
 
+BUILD_DATE=$(shell dpkg-parsechangelog -S Date -l../debian/changelog | LC_ALL=C date -u "+%d %B %Y" -f -)
 SGMLFILES=$(shell echo *.sgml)
 HTMLFILES=$(subst .sgml,.html,$(SGMLFILES)) index.html
 MANFILES=$(subst .sgml,.8,$(SGMLFILES))
@@ -43,7 +44,7 @@ iputils.html: iputils.db
 $(MANFILES): index.db
 	@set -e; \
 	$(SETUP_TMPDIR); \
-	nsgmls ../$< | sgmlspl ../docbook2man-spec.pl ""; \
+	nsgmls ../$< | sgmlspl ../docbook2man-spec.pl "$(BUILD_DATE)"; \
 	mv $@ ..; \
 	$(CLEAN_TMPDIR)
 
